controllers/status/status.go

```diff
@@ -56,10 +56,8 @@ const (
 )
 
 const (
-	// TODO: update this list of constants with proper reasons for conditions
 	ReconcileInitiatedReason = "ReconcileInitiated"
```

Comment:
seem this is not used anywhere, can remove?

Reply:
Done